package.json

```diff
@@ -26,7 +26,7 @@
     "backstage-create": "backstage-cli create --scope backstage --no-private",
     "create-plugin": "yarn backstage-create --select plugin",
     "remove-plugin": "backstage-cli remove-plugin",
-    "release": "node scripts/prepare-release.js && changeset version && yarn diff --yes && yarn prettier --write '{packages,plugins}/*/{package.json,CHANGELOG.md}' && yarn install",
+    "release": "node scripts/prepare-release.js && changeset version && yarn diff --yes && yarn prettier --write '{packages,plugins}/*/{package.json,CHANGELOG.md}' .changeset && yarn install",
```

Comment:
hmmm i think i'd prefer just running it on pre.json

Reply:
Why not? :grin: Dunno if it's possible that changesets decides to do something with those

Reply:
It's also really annoying because we'd need to check if the file exists :p

Reply:
eh ok then

Reply:
unless we can go *.json

Reply:
alright, doesn't hurt